<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentWindow</class>
 <widget class="QMainWindow" name="AgentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabBar::tab { 
	font: 500 8pt &quot;Montserrat Medium&quot;;
    background:	#e9e9e9;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
	height: 34 px;
	padding: 8px;
}

QTabBar::tab:selected {
    background-color: white;
}
</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="ConcludeTab">
       <attribute name="title">
        <string>Заключить договор</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="1">
           <widget class="PhoneEdit" name="Phone">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QLineEdit:focus {
    border-color: #5fbaeb;
}
</string>
            </property>
            <property name="maxLength">
             <number>18</number>
            </property>
            <property name="placeholderText">
             <string>Телефон</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="5">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="Patronymic">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QLineEdit:focus {
    border-color: #5fbaeb;
}
</string>
            </property>
            <property name="placeholderText">
             <string>Отчество</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QComboBox" name="TypeContract">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QComboBox:editable {
    background: #5fbaeb;
}

QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: #cdcdcd;
}

QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 25px;
	border-left-width: 1px;
}

QComboBox::down-arrow {
   image: url(:/images/resources/CheckMark.png);
	width: 12px;
	height: 12px;
}
QComboBox QAbstractItemView::item {
height: 40px;
}
</string>
            </property>
            <property name="placeholderText">
             <string>Выберите тип договора</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="4">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="2">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">QLabel {
	font: 500 11pt &quot;Montserrat Medium&quot;;
}
</string>
            </property>
            <property name="text">
             <string>Информация о клиенте:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="NumberEdit" name="Summa">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QLineEdit:focus {
    border-color: #5fbaeb;
}
</string>
            </property>
            <property name="placeholderText">
             <string>Сумма</string>
            </property>
           </widget>
          </item>
          <item row="7" column="5">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="ConcludeContractButton">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
font: 500 14pt &quot;Montserrat Medium&quot;;
    background-color: #4CAF50; 
    color: white; 
    border: none;
    font-size: 14px;
    border-radius: 5px; 
}
QPushButton:hover {
    background-color: #45a049;
}
QPushButton:pressed {
    background-color: #357a38; 
}</string>
              </property>
              <property name="text">
               <string>Заключить договор</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="LastName">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QLineEdit:focus {
    border-color: #5fbaeb;
}
</string>
            </property>
            <property name="placeholderText">
             <string>Фамилия</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">QLabel {
	font: 500 11pt &quot;Montserrat Medium&quot;;
}
</string>
            </property>
            <property name="text">
             <string>Информация о договоре:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="FirstName">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QLineEdit:focus {
    border-color: #5fbaeb;
}
</string>
            </property>
            <property name="placeholderText">
             <string>Имя</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3" rowspan="2">
           <widget class="QTextEdit" name="Description">
            <property name="styleSheet">
             <string notr="true">QTextEdit {
	font: 500 9pt &quot;Montserrat Medium&quot;;
   background: #e9e9e9;
	border-radius: 5px;
	padding: 8px;
	border: 1px solid gray;
}

QTextEdit:focus {
    border-color: #5fbaeb;
}</string>
            </property>
            <property name="placeholderText">
             <string>Описание объекта договора</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="RenegotiateContract">
       <attribute name="title">
        <string>Перезаключить договор</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QStackedWidget" name="stackedWidgetRenegotiateContract">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page"/>
          <widget class="QWidget" name="page_2"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="StatisticTab">
       <attribute name="title">
        <string>Просмотреть статистику</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTableView" name="statisticView">
          <property name="styleSheet">
           <string notr="true">QTableView {
    color: black; /*цвет текста внутри таблицы*/
    gridline-color: black /*цвет сетки*/;
}
QHeaderView::section {
	background-color: #e9e9e9; /*цвет названия солбцов и номеров строки*/
   color: black; /*цвет текста хэдеров*/
}

QScrollBar:horizontal 
{
   background-color: #e9e9e9;
	border: 1px solid  #7a7a7a;
}


QScrollBar::handle:horizontal 
{
    background-color: #aeaeae;
}

QScrollBar::add-line:horizontal, 
QScrollBar::sub-line:horizontal,
QScrollBar::add-page:horizontal, 
QScrollBar::sub-page:horizontal 
{
    width: 0px;
    background-color: transparent;
}


QScrollBar:vertical 
{
  	background-color: #e9e9e9;
	border: 1px solid  #7a7a7a;
}

QScrollBar::handle:vertical 
{
    background-color: #aeaeae;
}


QScrollBar::add-line:vertical, 
QScrollBar::sub-line:vertical,
QScrollBar::add-page:vertical, 
QScrollBar::sub-page:vertical 
{
    height: 0px;
    background-color: transparent;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhoneEdit</class>
   <extends>QLineEdit</extends>
   <header>phoneedit.h</header>
  </customwidget>
  <customwidget>
   <class>NumberEdit</class>
   <extends>QLineEdit</extends>
   <header>numberedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>LastName</tabstop>
  <tabstop>FirstName</tabstop>
  <tabstop>Patronymic</tabstop>
  <tabstop>Phone</tabstop>
  <tabstop>TypeContract</tabstop>
  <tabstop>Summa</tabstop>
  <tabstop>ConcludeContractButton</tabstop>
  <tabstop>statisticView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
